content/docs/install-pack.md

```diff
@@ -19,7 +19,7 @@ You can install the most recent version of `pack` (version **{{< latest >}}**) a
 To install `pack` on macOS, the easiest way is to use Homebrew:
 
 ```bash
-brew tap buildpack/tap
+brew tap buildpacks/tap
 brew install pack
```

Comment:
```suggestion
brew install buildpacks/tap/pack
```

Minor: We started doing this on release notes as a way to condense two commands into one. Probably worth doing here  as well.